src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -747,9 +766,39 @@
   should be copied to."
   ([cacrl :- schema/Str
     localcacrl :- schema/Str]
-    (when (fs/exists? cacrl)
-      (ks/mkdirs! (fs/parent localcacrl))
-      (fs/copy cacrl localcacrl))))
+    (when (not= cacrl localcacrl)
```

Comment:
This, while the same indentation level as the previous code, is also off by one.

Reply:
I don't see this one. Don't we want it to be two spaces inside the previous parenthesis level? And if only one, why?

Reply:
See the function above this one for the way it's supposed to look, this comes from
https://github.com/bbatsov/clojure-style-guide#multiple-arity-indentation

Basically, we want to arguments and body to line up.

However, neither this nor the one before it should have the additional parens around them. It looks like maybe both of these used to be multi-arity? Or maybe the style has changed? I think this is some pretty old code.

Sorry, I didn't originally expand the code above this to see that it *wasn't* a multi-arity function. I would recommend removing the set of parens around the arguments with that knowledge.

Reply:
Okay that makes sense.